docs/topics_5/proc_web-downloading-logs-cli.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide_5/master.adoc
+
+[id="proc_web-downloading-logs-cli_{context}"]
+= Downloading logs by using the CLI
+[role="_abstract"]
+
+You can download pod logs by using the CLI.
+
+.Procedure
+
+. Obtain the pod names:
++
+----
+$ oc get pods -n <project-name>
+----
++
+The output resembles the following:
++
+[source,terminal]
+----
+NAME                       READY     STATUS      RESTARTS   AGE
+eap-builder-1-build        0/1       Completed   0          1d
+mta-postgresql-1-hfbdn     1/1       Running     0          1d
+mta-sso-1-build            0/1       Completed   0          1d
+mta-web-console-1-build    0/1       Completed   0          1d
+mta-web-console-1-vt7s5    1/1       Running     1          1d
+sso-1-wjl2n                1/1       Running     1          1d
+----
+
+. Use `oc logs` to examine the pod log:
++
+----
+$ oc logs <pod>
+----
+
```

Comment:
Add a `+` to line 37. The note refers to the step, so it should be in line with the step.